packages/core/src/components/toast/toaster.tsx

```diff
@@ -89,6 +93,9 @@ export class Toaster extends AbstractComponent<IToasterProps, IToasterState> imp
      * The `Toaster` will be rendered into a new element appended to the given container.
      */
     public static create(props?: IToasterProps, container = document.body): IToaster {
+        if (props != null && props.inline != null) {
```

Comment:
sure, but it won't get flagged for you automatically; you'll need to look for all such comparisons against `null`. in my ideal world, we also enable the `no-null-keyword` lint rule and any explicit usage of `null` would have to be whitelisted